src/MovieCard/MovieCard.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { BrowserRouter, Route, Link } from 'react-router-dom';
 import './MovieCard.css';
 
+// function determineButton(props) {
+// }
+
+
 function MovieCard(props) {
+  const path = `/movie/${props.id}`;
+  console.log('card', props)
   return (
     <article className='movie-card'>
       <img className='movie-card-image' alt='single movie card for {props.title}' src={props.posterPath} />
       <h3 className='movie-title'>{ props.title }</h3>
       <p className='movie-average-rating'>Average Rating: { props.averageRating }</p>
+      <Link to={path}><button className='show-page-button'>Show Me More!</button></Link>
```

Comment:
I'm still confused about using `Link` here get to our corresponding `ShowPage` and using `Route` in `App` to go to the `ShowPage` based on `id` of the selected movie.  I'm wondering if these two pieces of code are working together, or if one is doing all the work.  We should test this out!